src/Azure.Functions.Cli/Helpers/PythonHelpers.cs

```diff
@@ -19,17 +18,27 @@ public static class PythonHelpers
         public static string VirtualEnvironmentPath => Environment.GetEnvironmentVariable("VIRTUAL_ENV");
         private static WorkerLanguageVersionInfo _pythonVersionCache = null;
 
-        public static async Task SetupPythonProject(bool generatePythonDocumentation = true)
+        public static async Task SetupPythonProject(ProgrammingModel programmingModel, bool generatePythonDocumentation = true)
         {
             var pyVersion = await GetEnvironmentPythonVersion();
             AssertPythonVersion(pyVersion, errorIfNoVersion: false);
 
+            // We print this message to the user irrespective of whether they choose the stable or preview programming model
+            // for awareness and reference purposes respectively
+            ColoredConsole.Write(AdditionalInfoColor("The PyStein Python programming model is in public preview. Learn more at "));
```

Comment:
Correct, PyStein is not an approved external name. Apologies @michaelpeng36 for not changing this message in the spec. Is it possible to have it say that "The new Python programming model..." instead of PyStein?